utils/contributors.js

```diff
@@ -59,6 +59,35 @@ const CONTRIBUTERS = [
         `,
         github: "https://github.com/debarati-06",
     },
+    
+    {
+        handle_name: 'Harshith1263',
+        name: "Harshith",
+        image: "/home/harshith/CSOC22-Week0-namecards/public/images/Harshith.jpeg",
+        description: `
+            <script src="https://cdn.lordicon.com//libs/frhvbuzj/lord-icon-2.0.2.js"></script>
+            <div class="text-center">
+               <lord-icon src="https://cdn.lordicon.com/ycnmpwon.json" trigger="loop" colors="primary:#121331,secondary:#66d7ee" style="width:25px;height:25px"></lord-icon>    
+                    Chilling
+                <lord-icon src="https://cdn.lordicon.com/ycnmpwon.json" trigger="loop" colors="primary:#121331,secondary:#66d7ee" style="width:25px;height:25px"></lord-icon>    
+                <br />
+                <lord-icon src="https://cdn.lordicon.com/wxnxiano.json" trigger="loop" colors="primary:#121331,secondary:#2516c7" style="width:25px;height:25px"></lord-icon>
+                    Working
+                <lord-icon src="https://cdn.lordicon.com/wxnxiano.json" trigger="loop" colors="primary:#121331,secondary:#2516c7" style="width:25px;height:25px"></lord-icon>
+                    <br />
+                <lord-icon src="https://cdn.lordicon.com/fvkdonkw.json" trigger="loop" colors="primary:#121331,secondary:#c7166f" style="width:25px;height:25px"></lord-icon>
+                    Playing
+                <lord-icon src="https://cdn.lordicon.com/fvkdonkw.json" trigger="loop" colors="primary:#121331,secondary:#c7166f" style="width:25px;height:25px"></lord-icon>
+                <br />
+                <lord-icon src="https://cdn.lordicon.com/jpdtnwas.json" trigger="loop" colors="primary:#121331,secondary:#e88c30" style="width:25px;height:25px"></lord-icon>
+                    Eating
+                    <lord-icon src="https://cdn.lordicon.com/jpdtnwas.json" trigger="loop" colors="primary:#121331,secondary:#e88c30" style="width:25px;height:25px"></lord-icon>
+            </div>
+        `,
+        github: "https://github.com/Harshith1263",
+    }
 ];
 
 export default CONTRIBUTERS;
+
+
```

Comment:
Remove these blank lines as well.

Reply:
Yeah its Now Resolved

Reply:
As all the things are Resolved Bhayya Can U please Give the Label Passed after checking !!!